course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -370,25 +370,61 @@
         $scope.selectRole = function(role) {
             $scope.selectedRole = role;
         };
-        $scope.setTitle = function(id) {
+        $scope.setCourseInstanceData = function(id) {
             var ci = courseInstances.instances[id];
             if (angular.isDefined(ci)) {
-                $scope.title = ci.title;
+                $scope.courseInstance = $scope.getFormattedCourseInstance(ci)
             }
             else {
-                $scope.title = '';
                 var url = djangoUrl.reverse(
                               'icommons_rest_api_proxy',
                               ['api/course/v2/course_instances/' + id + '/']);
                 $http.get(url)
                     .success(function(data, status, headers, config, statusText) {
                         courseInstances.instances[data.course_instance_id] = data;
-                        $scope.title = data.title;
+                        $scope.courseInstance = $scope.getFormattedCourseInstance(data)
+
                     })
                     .error($scope.handleAjaxError);
             }
         };
 
+        $scope.getFormattedCourseInstance = function(ci) {
+            courseInstance = {};
```

Comment:
Can we add a comment explaining that this is essentially a combination of the logic in `SearchController.courseInstanceToTable()` and the `SearchController` cell render functions?  Maybe add a note over there as well, so we remember to make changes in both places, or a TODO to split that logic out into a library eventually?